examples/nextjs/src/components/PageWrapper.tsx

```diff
@@ -1,42 +1,72 @@
 import { Header } from "./Header";
 import { AudioPlayer } from "@public-assembly/audio-player-ui";
-import useAudioPlayer from "../hooks/useAudioPlayer";
+import { useAudioPlayer } from "@public-assembly/audio-player-ui";
+import { useEffect, useRef } from "react";
+
+type MusicNFT = {
+  id: string;
+  artist: string;
+  title: string;
+  image: string;
+  audioSrc: string;
+};
+//FAKE DATA
+const INIT_TEST_MUSIC_NFT = {
+  id: "1",
+  artist: "Public Assembly",
+  title: "Test NFT",
+  image: "https://i.imgur.com/1Q9ZQ9r.png",
+  audioSrc:
+    "https://arena-attachments.s3.amazonaws.com/18170014/30a9e158e4001a12a1cb9e2c2f05586a.wav?1663852383",
+} as MusicNFT;
 
 export function PageWrapper({
   children,
   ...props
 }: {
   children?: JSX.Element;
 }) {
+  const trackRef = useRef<HTMLAudioElement>(null);
   const {
-    curTime,
-    duration,
-    playing,
-    setPlaying,
-    setClickedTime,
-    toggleMute,
-    muted,
+    onLoad,
+    onLoadedMetadata,
+    onPlay,
+    onPause,
+    onTimeUpdate,
+    setCurrentTrack,
   } = useAudioPlayer();
-  //HARDCODED FOR NOW
-  const artistName = "Artist";
-  const trackName = "Song";
+
+  useEffect(() => {
+    setCurrentTrack(trackRef.current);
+  }, [trackRef]);
+
   return (
     <>
       <Header />
       <main {...props} className="p-4">
+        <AudioPlayer
+          id={INIT_TEST_MUSIC_NFT.id}
+          artist={INIT_TEST_MUSIC_NFT.artist}
+          title={INIT_TEST_MUSIC_NFT.title}
+          image={INIT_TEST_MUSIC_NFT.image}
+          audioSrc={INIT_TEST_MUSIC_NFT.audioSrc}
+        />
+        <audio
```

Comment:
The audio element should be in the package component.
That way it's completely portable and does not require the user to do a bunch of additional setup on their end.

We could eventually structure the component in a way that allows a user to optionally pass in an external audio tag - or object or whatever but that would take some extra consideration.